nndt/datasets/dataset.py

```diff
@@ -27,7 +27,6 @@ def load(self) -> None:
                     assert _check_md5(z, self.hash)
                     _extract_7z_file(z, self.to_path)
                 except Exception as e:
```

Comment:
This is exception hiding, the most dangerous anti-pattern! **THIS MUST BE FIXED!**

Reply:
fixed